blog/package.json

```diff
@@ -9,7 +9,7 @@
     "analyze": "webpack --mode=production --env analyze",
     "lint": "eslint src --ext js,ts,tsx",
     "format": "prettier --write \"./**\"",
-    "test": "cross-env BABEL_ENV=test jest",
+    "test": "cross-env BABEL_ENV=test jest --passWithNoTests",
```

Comment:
Instead of force npm to pass the test, you just need to remove the test phase in CI.